addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -49,6 +49,8 @@ const WebsiteWysiwyg = Wysiwyg.extend({
      */
     start: function () {
         this.options.toolbarHandler = $('#web_editor-top-edit');
+        // Do not insert a paragraph after each column added by the column commands:
+        this.options.pAfterColumn = false;
```

Comment:
it is unclear to me when do we want this to be false ?

Reply:
In website. FP spec.

Reply:
I was unclear. From the changes in this PR, it seems this variable can only ever be `false` or `undefined`, which is why it feels useless to me. I guess it's missing somewhere where it is set to true then ?

Reply:
It's only for use in the `columnize` command, where its default value is `true`.
```js
columnize: (editor, numberOfColumns, addParagraphAfter=true) => {
```

Reply:
Wow that is kind of harsh... The only reason this works is because javascript cannot differentiate between a "true" `undefined` that is passed as parameter and a parameter that is actually undefined per se. I think it's weird to pass an actual parameter then rely on some quirks of javascripts to fall back to the default parameter as if none was passed. Could you please have this parameter set to true on the editor options somewhere ? This will be way more explicit in how it works (no js quirk) _and_ will make the existance of the option actually explicit. Currently, the only way to know it exists is to look at the code and notice that we access an options variable that seems to never be set to true but somehow the behavior is enabled regardless so... quite confusing right now.

Reply:
Sure I'll do that. I can see how that's unclear because you have to know the arguments of the command etc. But I don't know if I would say it's a quirk of Javascript. If the argument is not defined, a default value is applied. That's quite literally what default values are for isn't it?

Reply:
I consider this a quirk because `undefined` is an actual value that you can pass and that value is interpreted as if nothing was passed:
```js
f(undefined) === f()
```
I understand what you mean in the sense that this magic value _is_ actually called "undefined" so it does somewhat make sense. In other languages however, for example Python, where the equivalent of `undefined` is `None`, having this behavior would be really weird because then you couldn't rely on the name of the magical value to justify that behavior.
```python
f(None) !== f()
```

Reply:
> then you couldn't rely on the name of the magical value to justify that behavior

I'd argue that you could because `None` means `No` argument was passed. We passed none === we passed nothing === we passed no argument.

I get your point that it's just a question of naming. However, is it really?
```js
f(arg) {
    return arg === undefined;
}
console.log(f()); // true
console.log(f(undefined)); // true
```
It's also the value of anything that is... not defined. The fact that we can define something to be undefined is the weird thing about all this, not the fact that `undefined` means `nothing`.

I believe the same is true for Python. You'll test if a variable was previously defined by checking if it `is None`.

Reply:
Yes indeed that's what I said, but my original point was semantic, that in Javascript, `undefined` is also the default value of anything that was not explicitly defined.
But it's true indeed that that's specific to Javascript so in that sense we could say it's a "quirk" of Javascript.

Reply:
Now are you gonna r+ this shit or what? :-D

Reply:
Ok yes I get what you meant. Indeed it's not a "quirk" of Javascript in the sense that the behavior does makes sense considering the _other_ behaviors of Javascript, regardless of the name of the variable (though I would argue that it is named like that specifically because of the behaviors surrounding it), but it is a quirk of javascript compared to _other languages_, is what I meant... :p